apps/condo/domains/common/components/Prompt.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState, useRef } from 'react'
+import { Modal, Typography, FormInstance } from 'antd'
+import { useIntl } from '@core/next/intl'
+import { useRouter } from 'next/router'
+import { Button } from '@condo/domains/common/components/Button'
+
+interface IPromptProps {
+    title: string
+    form: FormInstance<unknown>
+    handleSave: () => void
+}
+
+const Prompt: React.FC<IPromptProps> = ({ children, title, form, handleSave: formSubmit }) => {
+    const [next, setNext] = useState(null)
+    const isIgnoringPrompt = useRef(false)
+    const [isModalVisible, setIsModalVisible] = useState(false)
+    const router = useRouter()
+    const intl = useIntl()
```

Comment:
It's better to use common style for components: https://github.com/open-condo-software/condo/blob/master/docs/style.md#react-component-style

intl should be the first

Reply:
Done